Classes/Scheduler/Base.php

```diff
@@ -96,7 +96,13 @@ public static function context(array $tsconf = [], array $conf = [], ?string $pi
 
         $context->setI18n(Aimeos\Base::i18n($langids, (array) ($tsconf['i18n'] ?? [])));
 
-        $tmplPaths = Aimeos\Base::aimeos()->getTemplatePaths('controller/jobs/templates', $context->locale()->getSiteItem()->getTheme());
+        try {
+            $theme = $context->locale()->getSiteItem()->getTheme();
+        } catch(\Exception $e) {
+            $theme = null;
+        }
```

Comment:
That doesn't work as there's never a locale item available so please remove all changes here completely. The job controllers look up the theme directly